.github/workflows/build-on-pull-request.yml

```
@@ -17,7 +17,7 @@ jobs:
         uses: actions/setup-java@v2
```

actionlint (1.7.4)

the runner of "actions/setup-java@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)